js/block.js

```diff
@@ -381,25 +381,25 @@ class Block {
         return false;
     }
 
-    offScreen(boundary) {
-        return !this.trash && boundary.offScreen(this.container.x, this.container.y);
-    }
+    //changed to arrow 
```

Comment:
No need to insert these comments.. they are implicit in the PR and not useful to someone reading the code later.